src/autonlp/evaluate.py

```diff
@@ -46,13 +18,10 @@ class Evaluate:
     @classmethod
     def from_json_resp(cls, json_resp: dict, token: str):
         task = get_task(json_resp["task"])
-        status_emoji, status = get_eval_job_status(json_resp["status"])
         return cls(
             _token=token,
             user=json_resp["username"],
             task=task,
-            status_emoji=status_emoji,
-            status=status,
```

Comment:
No status reporting anymore ?

Reply:
not really. i will add it with some status checking cli command in near future